app/components/Inspector/Screenshot.js

```diff
@@ -46,17 +51,25 @@ export default class Screenshot extends Component {
     // Recurse through the 'source' JSON and render a highlighter rect for each element
     const highlighterRects = [];
 
+    let highlighterXOffset = 0;
+    if (this.containerEl) {
+      const screenshotEl = this.containerEl.querySelector('img');
+      highlighterXOffset = screenshotEl.getBoundingClientRect().left -
+                           this.containerEl.getBoundingClientRect().left;
+    }
+
```

Comment:
I assume you had to do this after you changed the CSS from 100% width to 100% height?